docs/guides/vault/interact-with-contracts.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: 'Interact With Contracts'
+sidebar_position: 3
+---
+
+# Interact With Contracts
+
+Similar to our **[previous guide](./edit-vault-data.md)** on setting data on the Vault, in this guide we will be learning how to **interact with other contracts** through the Vault's execute function.
+
+## Interact with contracts
+
+Check the **[edit Vault data guide](./edit-vault-data.md)** to know how to get the needed variables (myVault, myUP, myVaultAddress).
+
+```typescript title="Interacting with other contracts through the vault"
+// encode the payload of the function we want to call 
+// could be a transfer function of a token, function on a Decentralized exchange, etc.
+const targetPayload = targetContract.methods
```

Comment:
Where is `targetContract` coming from?

Add the instantiation above so we know what it is ;)

Reply:
This `targetContract` is just a dummy variable to showcase that you can call any other contract using the execute function.
It's used the same way in previous guides (UniversalProfile- Interact with contracts).